MySaver/Resources/Raw/stores.json

```diff
@@ -1,57 +1,1447 @@
+        "Longitude": 25.3160625
+    },
+    {
+        "Name": "Iki",
+        "Address": "Žirmūnų g. 2, Vilnius",
+        "DefaultSchedule": "07:30-23:00",
+        "Latitude": 54.697312499999995,
+        "Longitude": 25.29881250000001
+    },
+    {
+        "Name": "Iki",
+        "Address": "Senasis Ukmergės kl. 8, Avižieniai",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.769812500000015,
+        "Longitude": 25.195312499999986
+    },
+    {
+        "Name": "Iki Express",
+        "Address": "Ulonų g. 2, Vilnius",
+        "DefaultSchedule": "08:00-20:00",
+        "Latitude": 54.70831249999999,
+        "Longitude": 25.290312499999985
+    },
+    {
+        "Name": "Iki Express",
+        "Address": "Kalvarijų g. 105, Vilnius",
+        "DefaultSchedule": "08:00-20:00",
+        "Latitude": 54.70843749999999,
+        "Longitude": 25.285812500000006
+    },
+    {
+        "Name": "Iki",
+        "Address": "Žalioji g. 22a, Vilnius",
+        "DefaultSchedule": "8:30-22:00",
+        "Latitude": 54.7308125,
+        "Longitude": 25.1839375
+    },
+    {
+        "Name": "Iki",
+        "Address": "Sėlių g. 54, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.69831250000003,
+        "Longitude": 25.247437499999975
+    },
+    {
+        "Name": "Iki",
+        "Address": "Molėtų g. 13-1, Riešė",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.781437499999996,
+        "Longitude": 25.273812500000005
+    },
+    {
+        "Name": "Iki",
+        "Address": "Priegliaus g. 1, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.709812499999984,
+        "Longitude": 25.187312500000004
+    },
+    {
+        "Name": "Iki",
+        "Address": "Karaliaučiaus g. 9D, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.70793749999997,
+        "Longitude": 25.174437500000025
+    },
+    {
+        "Name": "Iki",
+        "Address": "Saulėtekio al. 43, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.72518749999999,
+        "Longitude": 25.342437500000003
+    },
+    {
+        "Name": "Iki",
+        "Address": "Savanorių pr. 1, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.679312499999995,
+        "Longitude": 25.25756249999999
+    },
+    {
+        "Name": "Iki",
+        "Address": "Sodų g. 22-1, Vilnius",
+        "DefaultSchedule": "07:00-22:00",
+        "Latitude": 54.67031250000001,
+        "Longitude": 25.281937500000012
+    },
+    {
+        "Name": "Iki Express",
+        "Address": "Kalvarijų g. 3, Vilnius",
+        "DefaultSchedule": "08:00-22:00",
+        "Latitude": 54.69281249999999,
+        "Longitude": 25.281062500000004
+    },
+    {
+        "Name": "Iki",
+        "Address": "Švitrigailos g. 29, Vilnius",
+        "DefaultSchedule": "08:00-21:00",
+        "Schedule": {
+            "Sunday": "9a.m.–7p.m."
```

Comment:
There are a few places where the schedule is in am/pm, might be good to have all the times be the same 24hour format instead

Reply:
Fixed